scripts/shopify-tag.css

```diff
@@ -1,8 +1,8 @@
-hr {
+.hc-horizontal-rule {
     margin: 24px 0;
 }
 
-p {
+.hc-text {
```

Comment:
I think we should use scoped styling on p tag

Reply:
Removed usage of class on p and hr tag, and used descendant selectors.